### PR TITLE
Fix `path` in `.strangelogrc` being overwritten with default `changelog/`

### PR DESCRIPTION
**Pull request by leoselig**
This leads to the discontinued support for the (inofficial) `--directory` option

### CODE REVIEW

**Review by slayerpart — DISMISSED** [review]
LGTM!

File: test/specs/cli/add.spec.js [hunk 1]
```diff
@@ -38,4 +38,28 @@ describe('$ add', () => {
...
+          CLIButtons.ARROW_DOWN,
+          CLIButtons.ENTER,
+          'the description',
+          CLIButtons.ENTER
+        ]);
+
+      const persistedEntry = readSingleYAMLFileFromGlob(testProject.changelogPath, 'next/*.yml');
+
+      expect(persistedEntry.component).toEqual('comp1');
+      expect(persistedEntry.kind).toEqual('change');
```
Comment (Kadrian) [inline test/specs/cli/add.spec.js:65]: shouldn't `kind` be `"add"`?

File: .gitignore [hunk 2]
```diff
@@ -4,4 +4,7 @@
 /npm-debug.log
 
 # Build output
-/lib
\ No newline at end of file
+/lib
+
+# Test output
+/tmpTest
```
Comment (Kadrian) [inline .gitignore]: Did tests already save output there? Wondering where this camelCased folder name came from :-)

File: test/specs/cli/add.spec.js (hunk 1 above)
Comment (leoselig) [reply test/specs/cli/add.spec.js:65]: No 😅

**Review by klara-l — APPROVED** [review]

**Pull request merged**